src/simulate.jl

```diff
@@ -92,6 +159,7 @@ end
 
 MAP(sim::NestedFilterSimulation; kwargs...) = MAP(sim.fstate.model; kwargs...)
 
+# TODO: I don't remember what this is for!
```

Comment:
Do you remember @camillegontier?

Reply:
No, I do not; in the function I use to record the data, I call `map = MAP(sim.fstate.model)`

I run the scripts with and without this line, and it gave exactly the same results. We can safely remove it.

Reply:
I actually mean the line below the comment.